tf2onnx/graph.py

```diff
@@ -73,8 +74,10 @@ def attr(self):
     def attr_onnx(self):
         onnx_attrs = {}
         for a in self._attr.values():
-            if a.name in utils.ONNX_VALID_ATTRIBUTES:
-                onnx_attrs[a.name] = a
+            schema = get_schema(self.type, self.graph.opset)
+            if schema:
+                if a.name in schema.attributes:
```

Comment:
schema.has_attribute(name) or schema.contains_attribute(name)?
in case, we may change attributes type (e.g. a dict with name->type constraint?) in the future.